vars-ulm.cfg

```diff
@@ -16,142 +16,142 @@ inverse_flattening = 298.257223563
 # Of variable attributes, which are required, link to VIC docs, duplicative (but required by the CF spec)
 
 [Evap]
-units = kg/m^2s
+units = kg.m-2.s-1
 long_name = total net evaporation from surface and vegetation
 description = Total net evaporation from surface and vegetation
 standard_name = total_net_evaporation
```

Comment:
Are the `standard_name`s all CF compliant? That is what it says in the metadata, but have you made sure this is actually the case (if not we should not say that it is).

For CF metadata standard, see http://cfconventions.org/